modules/applications-create-using-cli-image.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * applications/creating-new-applications.adoc
+
+[id="applications-create-using-cli-image-{context}"]
+= Creating an application from an image
+
+You can deploy an application from an existing image. Images can come from
+imagestreams in the {product-title} server, images in a specific registry, or
+images in the local Docker server.
+
+The `new-app` command attempts to determine the type of image specified in the
+arguments passed to it. However, you can explicitly tell `new-app` whether the
+image is a container image (using the `--docker-image` argument) or an imagestream
+(using the `-i|--image` argument).
+
+[NOTE]
+====
+If you specify an image from your local Docker repository, you must ensure that
+the same image is available to the {product-title} cluster nodes.
+====
+
+.Procedure
+
+* Create an application from the DockerHub MySQL image, for example:
++
+----
+$ oc new-app mysql
+----
+
+* Create an application using an image in a private registry, specify the full
+container image specification:
++
+----
+$ oc new-app myregistry:5000/example/myimage
+----
+
+
+////
+Commenting out because the registry should be secure.
+
+[NOTE]
+====
+If the registry containing the image is not
+ifdef::openshift-enterprise,openshift-origin[]
+xref:../../install_config/registry/securing_and_exposing_registry.adoc#securing-the-registry[secured
+with SSL],
+endif::[]
+ifdef::openshift-dedicated[]
+secured with SSL,
+endif::[]
+cluster administrators must ensure that the Docker daemon on the {product-title}
```

Comment:
In 4.x we did not have docker, so will not use docker domain.

Reply:
Thanks! I had it commented out, but am going ahead and removing it.